seat.c

```diff
@@ -82,7 +82,7 @@ static void keyboard_modifiers(void *data, struct wl_keyboard *wl_keyboard,
 	xkb_state_update_mask(state->xkb.state,
 		mods_depressed, mods_latched, mods_locked, 0, 0, group);
 	int caps_lock = xkb_state_mod_name_is_active(state->xkb.state,
-		XKB_MOD_NAME_CAPS, XKB_STATE_MODS_LOCKED);
+		XKB_MOD_NAME_CAPS, XKB_STATE_MODS_EFFECTIVE);
```

Comment:
Is there an issue with LOCKED?

Reply:
No issue. I just thought that EFFECTIVE would be more logical here.

I can remove this commit if you think that this is wrong.